apps/frontend/package.json

```diff
@@ -28,12 +28,11 @@
     "@types/react": "^18.2.73",
     "@types/react-dom": "^18.2.23",
     "@types/react-helmet": "^6.1.11",
+    "@vitejs/plugin-react": "^4.2.1",
     "ariakit": "^2.0.0-next.44",
-    "assets-webpack-plugin": "^7.1.1",
     "autoprefixer": "^10.4.19",
     "axios": "^1.6.8",
     "clsx": "^2.1.0",
-    "core-js": "^3.36.1",
     "css-loader": "^6.10.0",
```

Comment:
Still needed?

Reply:
Removed in another release thanks to [Knip](https://knip.dev/).